### DISCUSSION TITLE
Linux server + PostGreSQL migration

### DISCUSSION THREAD

**Discussion by RodGaiotto** [requarks/wiki#3950] [Help / Questions]
Hello folks,

I have a Linux server with the Wiki.JS 2.x standard installation. I'll need to migrate it to a new server, I would like to know if any one of you have any experience migrating the Wiki and preserving the users, groups and their documents content 100% safe.

The documentation available refers only to Wiki.js implementation with Docker. I'm wondering if it would work in a standard one:

Backup:
pg_dump wiki -U wiki -F c > wikibackup.dump

Restore:
cat ~/wikibackup.dump | pg_restore -U wiki -d wiki

or

pg_restore -U wiki -d wiki < wikibackup.dump

If you could share any migration experience I would appreciate that!

Thank you very much!

**Comment by NGPixel** [comment]
Yes, that would work. You only need to dump and then restore the database, whether its via the CLI or a UI like pgadmin.
